lib/logger/utils.ts

```diff
@@ -92,14 +92,17 @@ function sanitizeValue(value: any, seen = new WeakMap()): any {
   return valueType === 'string' ? sanitize(value) : value;
 }
 
-// TODO
-// eslint-disable-next-lint @typescript-eslint/explicit-module-boundary-types
-export function withSanitizer(streamConfig): bunyan.Stream {
+type BunyanStream = (NodeJS.WritableStream | Stream) & {
```

Comment:
Looks little strange, but I'm ok if it improves types

Reply:
Yes, those types are tricky and it's hard to do something with it